### ISSUE TITLE
Full disk encryption with Argon2id for /boot

### ISSUE THREAD

**Issue by nakomoto** [XeroLinuxDev/calamares-cfg#1]
Hello,

does your ISO support booting from an encrypted /boot with Lusk2+Argon2id?

**Comment by DarkXero-dev** [comment]
@nakomoto 

I don't think so. If you check code you will see it only supports luks1. Also never heard of the Argon2id thing...
